docs/docs/recipes.md

```diff
@@ -985,6 +985,139 @@ export const pageQuery = graphql`
 - [Guide to creating pages from data programmatically](/docs/programmatically-create-pages-from-data/)
 - [Example repo](https://github.com/gatsbyjs/gatsby/tree/master/examples/recipe-sourcing-markdown) for this recipe
 
+### Sourcing from WordPress
+
+#### Prerequisites
+
+- An existing [Gatsby site](/docs/quick-start/) with a `gatsby-config.js` and `gatsby-node.js` file
+- A WordPress instance, either self-hosted or on Wordpress.com
+
+#### Directions
+
+1. Install the `gatsby-source-wordpress` plugin by running the following command:
+
+```shell
+npm install gatsby-source-wordpress --save
+```
+
+2. Configure the plugin by modifying the `gatsby-config.js` file such that it includes the following:
+
+```JS:title=gatsby-config.js
+module.exports = {
+  ...
+  plugins: [
+    {
+      resolve: `gatsby-source-wordpress`,
+      options: {
+        // baseUrl will need to be updated with your wordpress source
+        baseUrl: `wpexample.com`,
+        protocol: `https`,
+        // is it hosted on wordpress.com, or self-hosted?
+        hostingWPCOM: false,
+        // does your site use the Advanced Custom Fields Plugin?
+        useACF: false
+      }
+    },
+  ]
+}
+```
+
+> **Note:** Refer to the [`gatsby-source-wordpress` plugin docs](/packages/gatsby-source-wordpress/?=wordpre#how-to-use) to know more about configuring your plugins.
+
+3. Create a template component such as `src/templates/post.js` with the following code in it:
+
+```JS:title=post.js
+import React, { Component } from "react"
+import { graphql } from "gatsby"
+import PropTypes from "prop-types"
+
+class Post extends Component {
+  render() {
+    const post = this.props.data.wordpressPost
+
+    return (
+      <>
+        <h1>{post.title}</h1>
+        <div>{post.content}</div>
+      </>
+    )
+  }
+}
+
+Post.propTypes = {
+  data: PropTypes.object.isRequired,
+  edges: PropTypes.array,
+}
+
+export default Post
+
+export const pageQuery = graphql`
+  query($id: String!) {
+    wordpressPost(id: { eq: $id }) {
+      title
+      content
+    site {
```

Comment:
Since this section of the query isn't used (site/siteMetadata/title/subtitle) I think it could be used. It would also break in sites that this is attempted on that don't have a title and subtitle defined in that shape in a `gatsby-config`